doc/api/modules.md

```diff
@@ -352,22 +352,22 @@ If this was in a folder at `./some-library`, then
 
 This is the extent of Node.js's awareness of `package.json` files.
 
-If the file specified by the `'main'` entry of `package.json` is missing and
-can not be resolved, Node.js will report the entire module as missing with the
-default error:
-
-```txt
-Error: Cannot find module 'some-library'
-```
-
-If there is no `package.json` file present in the directory, then Node.js
+If there is no `package.json` file present in the directory, or if the
```

Comment:
I don’t think this change is correct; if package.json is present, even if “main” is missing or unresolvable, i don’t think it will fall back to index resolution.